### ISSUE TITLE
Encoding error: Stream did not contain valid UTF-8

### ISSUE THREAD

**Issue by StefanYohansson** [spk/maman#1]
I am trying to request from this site: `http://www2.defesasocial.rn.gov.br/estatistica/index.php`

This site is a mess, I think it is using ISO-8859-1.

Really appreciate if someone could help me to find a way to fix it.
Maybe rust-encoding could help: https://github.com/lifthrasiir/rust-encoding

`REDIS_URL="redis://127.0.0.1/" maman http://www2.defesasocial.rn.gov.br/estatistica/index.php 0`

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Custom(Custom { kind: InvalidData, error: StringError("stream did not contain valid UTF-8") }) }', ../src/libcore/result.rs:788
stack backtrace:
   1:     0x559497e6231f - std::sys::backtrace::tracing::imp::write::h46e546df6e4e4fe6
   2:     0x559497e65d7b - std::panicking::default_hook::_$u7b$$u7b$closure$u7d$$u7d$::h077deeda8b799591
   3:     0x559497e659ff - std::panicking::default_hook::heb8b6fd640571a4f
   4:     0x559497e592ce - std::panicking::rust_panic_with_hook::hd7b83626099d3416
   5:     0x559497e65fc1 - std::panicking::begin_panic::h941ea76fc945d925
   6:     0x559497e59f7a - std::panicking::begin_panic_fmt::h30280d4dd3f149f5
   7:     0x559497e65f5e - rust_begin_unwind
   8:     0x559497e9bfff - core::panicking::panic_fmt::h2d3cc8234dde51b4
   9:     0x559497e13a90 - core::result::unwrap_failed::h77e8f7274d900d66
  10:     0x559497e12d86 - robotparser::RobotFileParser::read::h06a4d8586bda24d1
  11:     0x559497dd31ef - maman::maman::Spider::crawl::h420ed92aa35e46c5
  12:     0x559497dca44f - maman::main::h6af72f80c0d7c8a7
  13:     0x559497e65608 - std::panicking::try::call::hca715a47aa047c49
  14:     0x559497e6d5db - __rust_try
  15:     0x559497e6d57e - __rust_maybe_catch_panic
  16:     0x559497e650ae - std::rt::lang_start::h162055cb2e4b9fe7
  17:     0x7f139b98682f - __libc_start_main
  18:     0x559497dc9fe8 - _start
  19:                0x0 - <unknown>
```

**Comment by spk** [comment]
Hello, thanks for the report !
Its something I [know](https://github.com/spk/maman/blob/master/src/maman/mod.rs#L255)... I will work on that.

**Comment by spk** [comment]
Released in v0.6.0
